laletterbuilder/forms.py

```diff
@@ -2,9 +2,20 @@
 from loc import models as loc_models
 
 from loc.forms import validate_non_stupid_name
+from onboarding.models import OnboardingInfo
+from project.forms import SetPasswordForm
 from project.util import lob_api
 
 
+class CreateAccount(SetPasswordForm, forms.ModelForm):
+    class Meta:
+        model = OnboardingInfo
+        fields = ("can_we_sms", "email")
```

Comment:
we might want to take the "email" out of here since it's not actually in onboarding info